### ISSUE TITLE
Cannot open include file: 'json.h': No such file or directory

### ISSUE THREAD

**Issue by espdev** [bgrimstad/splinter#83]
I'm trying to use your fine library, but I'm getting an error:
```
Cannot open include file: 'json.h': No such file or directory
```
I think this error is occurred because _thirdparty_ header file `json.h` has not been copied to include directory in cmake_install_prefix while INSTALL step.

I think you need to create a cmake config for standard CMake [find_package](https://cmake.org/cmake/help/v3.4/command/find_package.html) mechanism for using your library as external dependency in other cmake-based projects. For example:
```cmake
find_package(Splinter REQUIRED)

add_executable(MySplinesApp main.cpp)
target_include_directories(MySplinesApp ${SPLINTER_INCLUDE_DIRS})
target_link_libraries(MySplinesApp ${SPLINTER_LIBRARIES})
```

Thanks!

**Comment by gablank** [comment]
Thanks for your feedback. I agree, this is definitely something we should do. We are quite busy at the moment, but in the fall we intend to overhaul the library.

If you don't mind me asking, what do you use the library for, and what language do you use?

**Comment by espdev** [comment]
@gablank Thank you for your answer

I use the library for the interpolation and smoothing 1D and 2D data sets in our application which processes 2D/3D medical images. We use C++ as the main language and Python to develop prototypes and algorithms.

**Comment by bgrimstad** [comment]
Please verify that 8b27508 fixes this problem.

**Comment by gablank** [comment]
It seems like it does. Fixed in 8b2750849cee6a83e0e89a4045c1de2ff1b7597a.
